zerobuf/Zerobuf.h

```diff
@@ -10,6 +10,7 @@
 #include <zerobuf/api.h>
 #include <zerobuf/Types.h>
 #include <servus/uint128_t.h>
+#include <zerobuf/jsoncpp/json/json.h>
```

Comment:
toJSON returns JSON as string. Converting this to Json::Value outside of ZeroBuf should be possible.